scripts/run-build.ps1

```diff
@@ -44,7 +46,7 @@ $env:CHANNEL=$env:RELEASE_SUFFIX
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$PSScriptRoot\..\.dotnet_stage0\Windows"
+    $env:DOTNET_INSTALL_DIR="$PSScriptRoot\..\.dotnet_stage0\Windows\$Architecture"
```

Comment:
Totally missed that change, my bad.

sure makes sense for Windows only on the architecture.